docs/release_process.rst

```diff
@@ -3,21 +3,21 @@
 Release process
 ===============
 
-These instructions follow the hypothetical release of Kolibri version '0.3.0'.
+We maintain a set of `release process automation scripts <https://github.com/learningequality/kolibri-release-process/>`__ which automatically create internal next actions in `Notion <https://www.notion.so/learningequality/>`__. The processes there have separate sets of actions for:
 
-In this case, the repo would currently have a ``develop`` branch and a number of pre-existing release branches, the most recent being ``release-v0.2.x`` with potentially multiple tags for patch releases, e.g. ``v0.2.0`` and ``v0.2.1``.
-
-Patch releases (e.g. '0.3.1') follow the same process outlined below, except that development occurs exclusively on an existing release branch (e.g. ``release-v0.2.x``). Patch releases generally do not have new user-facing strings, so translation- and font-related steps can be skipped
+* finals, beta, and release candidates
+* major versus patch releases
 
+Branches and tags
+-----------------
 
-Create alpha releases
----------------------
+The examples below follow the hypothetical release of Kolibri version '0.3.0'. In this case, recent unstable changes would be in the ``develop`` branch, and there would be a number of pre-existing release branches up to ``release-v0.2.x``. Each would have tags for patch releases, e.g. ``v0.2.0`` and ``v0.2.1``.
 
 For the majority of development on a new release, PRs will target the ``develop`` branch. Tag alpha releases as desired using Github's `Releases <https://github.com/learningequality/kolibri/releases>`__ functionality, which both adds a tag to the git repo and creates a placeholder on the Github site with built distributions.
 
 Make sure to target ``develop``, use the standard tag naming convention (e.g. ``v0.3.0-alpha1``), and mark it has a "pre-release".
 
-These alphas can be used for preliminary testing as major, unstable updates are introduced.
+.. note:: The latest tagged releases are automatically deployed to `kolibri-dev.learningequality.org <https://kolibri-dev.learningequality.org/>`__.
```

Comment:
I think `kolibri-dev` contains the latest commit from develop branch, according to the description of the project: https://www.notion.so/learningequality/Continuous-deployment-for-Kolibri-9385f26adbcd41f7bcfe36c314cb1978

Reply:
updated, thanks